platform-hub-api/app/models/kubernetes_token.rb

```diff
@@ -0,0 +1,27 @@
+class KubernetesToken
+  include ActiveModel::Model
+  include ActiveModel::Serialization
+  include ActiveModel::Associations
+
+  CLUSTERS = [
```

Comment:
I don't think it's currently possible to use enum in ActiveModel. Moving the list over to hash record as discussed. We can refactor down the line.